ci/requirements-2.7_WIN.run

```diff
@@ -16,3 +16,4 @@ bottleneck
 html5lib
 beautiful-soup
 jinja2=2.8
```

Comment:
remove; we don't add to .run AND .pip

Reply:
remove the boto3 dependency? This file doesn't have moto in it currently.

I got an error claiming boto3 wasn't installed, but the file had s3fs; I'm using boto to interact with the moto objects to create the fake bucket and keys so I added boto3 to this run file.

Reply:
That's strange that boto3 didn't get installed, since both s3fs and moto depend on it. Do you have a link the logs of a CI run that failed without it?

Reply:
Either way, moto should be on conda-forge now, so you may be able to remove it from the pip files, and just have it in the regular conda one.

Reply:
@TomAugspurger just reviewed it; I'm mistaken. It was the requirements-3.6 windows run that did not have s3fs installed. I had added tests to the test_excel and read the wrong python version that had failed. So, I'll add moto to this run file and remove it from python-3.6 since that doesn't have s3fs in it. Sound good?

Reply:
Yep I think so.